samples/bot-daily-task-reminder/csharp/README.md

```diff
@@ -44,6 +44,20 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
 
 - [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
 
+- [Teams Toolkit for Visual Studio](https://learn.microsoft.com/en-us/microsoftteams/platform/toolkit/toolkit-v4/install-teams-toolkit-vs?pivots=visual-studio-v17-7)
+
+## Run the app (Using Teams Toolkit for Visual Studio)
+
+The simplest way to run this sample in Teams is to use Teams Toolkit for Visual Studio.
+1. Install Visual Studio 2022 **Version 17.9 or higher** [Visual Studio](https://visualstudio.microsoft.com/downloads/)
```

Comment:
For all the bot projects, need to be `Visual Studio 2022 version 17.9 Preview 2 or higher`

Reply:
Fixed